vmpool/platforms.py

```diff
@@ -6,6 +6,8 @@
 from core.logger import log_pool
 from core.utils import openstack_utils
 
+UnlimitedCount = type("UnlimitedCount", (), {})()
```

Comment:
@sh0ked As mentioned, it turns out that type has a totally separate use, when called with three arguments. type(name, bases, dict) creates a new type, programmatically.